editor/editor_inspector.cpp

```diff
@@ -405,6 +405,17 @@ void EditorProperty::_notification(int p_what) {
 				delete_rect = Rect2();
 			}
 		} break;
+		case NOTIFICATION_ENTER_TREE: {
+			if (has_borders) {
+				get_parent()->connect("theme_changed", callable_mp(this, &EditorProperty::_update_property_bg));
+				_update_property_bg();
+			}
+		} break;
+		case NOTIFICATION_EXIT_TREE: {
+			if (has_borders) {
+				get_parent()->disconnect("theme_changed", callable_mp(this, &EditorProperty::_update_property_bg));
```

Comment:
```suggestion
				get_parent()->connect(SNAME("theme_changed"), callable_mp(this, &EditorProperty::_update_property_bg));
				_update_property_bg();
			}
		} break;
		case NOTIFICATION_EXIT_TREE: {
			if (has_borders) {
				get_parent()->disconnect(SNAME("theme_changed"), callable_mp(this, &EditorProperty::_update_property_bg));
```

Reply:
I'm assuming this is what you meant as the code you suggested didn't seem to work `SceneStringNames::get_singleton()->theme_changed`. Please correct me if this was not what you were suggesting.

Reply:
No I suggested exactly what I suggested, it requires a rebase

Reply:
ok it was merged two hours ago that is why I didn't have it. Thanks for clarifying.